src/base/rss/utorrentrssdata.cpp

```diff
@@ -0,0 +1,190 @@
+#include "utorrentrssdata.h"
+
+#include <libtorrent/lazy_entry.hpp>
+
+#include <QDebug>
+#include <QFile>
+#include <QStringList>
+
+#if (QT_VERSION_MAJOR >= 5)
+#include <QRegularExpression>
+#else
+#include <QRegExp>
+#endif
+
+#include "base/rss/rssfeed.h"
```

Comment:
Since this file is in "base/rss" directory you should omit it in include statements.